.github/workflows/cloud-controller-build.yaml

```diff
@@ -60,6 +56,7 @@ jobs:
     needs: [ generate-matrix ]
     strategy:
       matrix: ${{ fromJson( needs.generate-matrix.outputs.matrix ) }}
+      fail-fast: false
```

Comment:
Add a comment for line 75. Not sure if this repo is visible and open source?

Reply:
open it in incognito mode to figure it out?